src/core/etl/src/Flow/ETL/Function/ScalarFunctionChain.php

```diff
@@ -487,6 +487,11 @@ public function stringAfter(ScalarFunction|string $needle, ScalarFunction|bool $
         return new StringAfter($this, $needle, $includeNeedle);
     }
 
+    public function stringAfterLast(ScalarFunction|string $needle, ScalarFunction|bool $includeNeedle = false) : self
```

Comment:
Could you start adding docblocks to those methods so it would be easier to understand what they are doing? 

Something like this: 

```
/**
 * Returns the contents found after the last occurrence of the given string
 **/ 
```